.storybook/main.ts

```diff
@@ -16,14 +16,31 @@
  * limitations under the License.
  */
 
-module.exports = {
+import type { StorybookConfig } from '@storybook/react-webpack5';
+
+const config: StorybookConfig = {
   stories: ['../examples/**/stories.@(ts|tsx|js|jsx)'],
   addons: [
     '@storybook/addon-a11y',
     '@storybook/addon-actions',
     '@storybook/addon-viewport',
+    '@storybook/addon-docs',
   ],
-  core: {
-    builder: 'webpack5',
+  framework: {
+    name: '@storybook/react-webpack5',
+    options: {},
```

Comment:
Does this require to be provided even when it's empty?

Reply:
I'm not sure, it was added automatically by the migration script